### ISSUE TITLE
esp missing databyte calculation

### ISSUE THREAD

**Issue by bf31415** [arkime/arkime#1116]
Elasticsearch version:

6.7.1

Moloch version:

2.0

OS name and version:

ubuntu 16.04

How was Moloch built/installed: (rpm, deb, easybutton, ...)

easybutton

Provide logs, stack traces and steps to reproduce:

the databytes calculation is missing for esp.  created separate issue in order to allow icmp databyte calculation discussion to be seperate from how esp databyte calc might be performed.

relevant code showing the issue in moloch_packet_process ()
```
case IPPROTO_UDP:
    udphdr = (struct udphdr *)(packet->pkt + packet->payloadOffset);
    packet->direction = (dir &&
                         session->port1 == ntohs(udphdr->uh_sport) &&
                         session->port2 == ntohs(udphdr->uh_dport))?0:1;
    session->databytes[packet->direction] += (packet->pktlen - 8);
    break;
case IPPROTO_SCTP:
    udphdr = (struct udphdr *)(packet->pkt + packet->payloadOffset);
    packet->direction = (dir &&
                         session->port1 == ntohs(udphdr->uh_sport) &&
                         session->port2 == ntohs(udphdr->uh_dport))?0:1;
    session->databytes[packet->direction] += (packet->pktlen - 8);
    break;
[snip]

case IPPROTO_ICMP:
case IPPROTO_ESP:
    packet->direction = (dir)?0:1;
    break;
}```

**Comment by awick** [comment]
Currently we don't do any ESP processing really, so its more than just this missing.  The original ESP request was just to allow downloading.

**Comment by awick** [comment]
fixed in #2634
